lib/danger/helpers/comments_helper.rb

```diff
@@ -1,10 +1,11 @@
-require "redcarpet"
+# require "redcarpet"
+require "kramdown"
```

Comment:
Cool, yeah, this looks good - can you remove this moment then we're good to go 👍